README.md

```diff
@@ -9,6 +9,14 @@ The synchronization works like this
 * creating a new OpenShift Build for a BuildConfig associated with a Jenkins Job results in the Jenkins Job being triggered
 * changes in a Jenkins Build Run thats associated with a Jenkins Job gets replicated to an OpenShift Build object (which is created if necessary if the build was triggered via Jenkins)
 
+Configuration
+------------------------
+Jenkins Build Log URL: 
+* The plugin adds an annotation in Build resource metadata which contains URL to access Jenkins build logs.
```

Comment:
This plugin adds an annotation to the OpenShift build configuration containing the Jenkins build log URL.